roles/core/pxe_stack/molecule/default/prepare.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Prepare
+  hosts: management1
+  tasks:
+    - name: Install firewall package
+      package:
+        name: firewalld
+        state: present
+
+    - name: Start firewall
+      service:
+        name: firewalld
+        enabled: yes
+        state: started
+
+    - name: "Configure BlueBanquise repository"
+      yum_repository:
+        name: bluebanquise
+        description: BlueBanquise 1.3 repository
+        baseurl: https://bluebanquise.com/repository/releases/1.3/el$releasever/$basearch/bluebanquise/
+        gpgcheck: False
+      when:
+        - ansible_facts.os_family == "RedHat"
+        - ansible_facts.distribution_major_version == "8"
```

Comment:
You can use 1.3 for el7 as well (since today). 😄 
You can even use https://bluebanquise.com/repository/releases/latest/

Reply:
Perfect !
The packages for el7 exist, however the repository is not yet created.
I believe that a `creatrepo . `is missing in order to have a repository under 
 [https://bluebanquise.com/repository/releases/latest/el7/x86_64/bluebanquise/](https://bluebanquise.com/repository/releases/latest/el7/x86_64/bluebanquise/)

@oxedions could you create the repo please ?

Reply:
Yes, I will do it tomorrow afternoon. It is planned 😊 I forgot to create the repo, my apologies.